core/src/main/scala/com/carto/analyticstoolbox/spark/sql/SpatialFilterPushdownOptimizations.scala

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2022 Azavea
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.carto.analyticstoolbox.spark.sql
+
+import org.apache.spark.sql.SparkSessionExtensions
+
+import com.carto.analyticstoolbox.spark.rules.SpatialFilterPushdownRules
```

Comment:
Could you also move `com.carto.analyticstoolbox.spark.rules.SpatialFilterPushdownRules` => `com.carto.analyticstoolbox.spark.sql.rules.SpatialFilterPushdownRules`?

Reply:
@moradology could you also move the `rules` package into `sql`?